tests/smoke/multi_cluster_deploy_test.go

```diff
@@ -0,0 +1,217 @@
+package smoke
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rook/rook/tests/framework/clients"
+	"github.com/rook/rook/tests/framework/enums"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	utilversion "k8s.io/kubernetes/pkg/util/version"
+)
+
+func TestInstallingMultipleRookCluster(t *testing.T) {
+	suite.Run(t, new(MultiRookClusterDeploySuite))
+}
+
+type MultiRookClusterDeploySuite struct {
+	suite.Suite
+	helper1     *clients.TestClient
+	helper2     *clients.TestClient
+	k8sh        *utils.K8sHelper
+	installer   *installer.InstallHelper
+	installData *installer.InstallData
+}
+
+//Deploy Multiple Rook clusters
+func (mrc *MultiRookClusterDeploySuite) SetupSuite() {
```

Comment:
for go conventions i'd recommend a single letter for the var name

Reply:
i am trying to use single letters where ever I can, but its hard to use single letter when there are multiple test suites in the same package. And I didn't want to pick random letters.